packages/pds/src/account-manager/index.ts

```diff
@@ -211,15 +211,21 @@ export class AccountManager
   async createSession(
     did: string,
     appPassword: password.AppPassDescript | null,
+    isSoftDeleted = false,
   ) {
     const { accessJwt, refreshJwt } = await auth.createTokens({
       did,
       jwtKey: this.jwtKey,
       serviceDid: this.serviceDid,
-      scope: auth.formatScope(appPassword),
+      scope: isSoftDeleted
+        ? AuthScope.Takendown
+        : auth.formatScope(appPassword),
```

Comment:
can we pass through `isSoftDeleted` to `formatScope` instead of doing the ternary logic here?